hooks/useStore.js

```diff
@@ -0,0 +1,16 @@
+import create from "zustand";
+
+export const useStore = create((set) => ({
+  articles: [],
+  article: {},
```

Comment:
maybe rename this to currentArticle, since this will be the one article that is presented, right?

Reply:
yep u re right!